<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-Q37MKND" name="com.bitnei.apitest.testcases.saas.PostBackSaas" tests="7" failures="5" timestamp="02 六月 2020 08:22:45 GMT" time="1.431" errors="0">
  <testcase name="UserInfoQueryData" time="0.463" classname="com.bitnei.apitest.testcases.saas.PostBackSaas">
    <failure type="java.lang.AssertionError" message="expected [{}] but found [{&quot;size&quot;:{&quot;currentValue is &quot;:0,&quot;expectedValue is&quot;:1},&quot;pages&quot;:{&quot;currentValue is &quot;:0,&quot;expectedValue is&quot;:1},&quot;total&quot;:{&quot;currentValue is &quot;:0,&quot;expectedValue is&quot;:1},&quot;Length&quot;:{&quot;actualLenth&quot;:0,&quot;expectLenth&quot;:1}}]">
      <![CDATA[java.lang.AssertionError: expected [{}] but found [{"size":{"currentValue is ":0,"expectedValue is":1},"pages":{"currentValue is ":0,"expectedValue is":1},"total":{"currentValue is ":0,"expectedValue is":1},"Length":{"actualLenth":0,"expectLenth":1}}]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.bitnei.apitest.testcases.saas.PostBackSaas.UserInfoQueryData(PostBackSaas.java:75)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- UserInfoQueryData -->
  <testcase name="UserInfoQueryData" time="0.119" classname="com.bitnei.apitest.testcases.saas.PostBackSaas">
    <failure type="java.lang.AssertionError" message="expected [{}] but found [{&quot;size&quot;:{&quot;currentValue is &quot;:0,&quot;expectedValue is&quot;:1},&quot;pages&quot;:{&quot;currentValue is &quot;:0,&quot;expectedValue is&quot;:1},&quot;total&quot;:{&quot;currentValue is &quot;:0,&quot;expectedValue is&quot;:1},&quot;Length&quot;:{&quot;actualLenth&quot;:0,&quot;expectLenth&quot;:1}}]">
      <![CDATA[java.lang.AssertionError: expected [{}] but found [{"size":{"currentValue is ":0,"expectedValue is":1},"pages":{"currentValue is ":0,"expectedValue is":1},"total":{"currentValue is ":0,"expectedValue is":1},"Length":{"actualLenth":0,"expectLenth":1}}]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.bitnei.apitest.testcases.saas.PostBackSaas.UserInfoQueryData(PostBackSaas.java:75)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- UserInfoQueryData -->
  <testcase name="UserInfoQueryData" time="0.212" classname="com.bitnei.apitest.testcases.saas.PostBackSaas">
    <failure type="java.lang.AssertionError" message="expected [{}] but found [{&quot;id&quot;:{&quot;currentValue is &quot;:&quot;96e1e2dd22b744e4978d76ac2b3f8cd3&quot;,&quot;expectedValue is &quot;:&quot;aa2a8a54936542b48e78346cb4f0df33&quot;},&quot;organization&quot;:{&quot;currentValue is &quot;:&quot;北京理工新源信息有限公司&quot;,&quot;expectedValue is &quot;:&quot;新源科技&quot;},&quot;tenant&quot;:{&quot;currentValue is &quot;:&quot;bitnei-cn&quot;,&quot;expectedValue is &quot;:&quot;bitnei-cn110222&quot;},&quot;connectMobile&quot;:{&quot;currentValue is &quot;:&quot;18945695263&quot;,&quot;expectedValue is &quot;:&quot;18110024813&quot;},&quot;origin&quot;:{&quot;currentValue is &quot;:&quot;REGISTER_FREE&quot;,&quot;expectedValue is &quot;:&quot;BACKEND_ADD&quot;},&quot;userNum&quot;:{&quot;currentValue is &quot;:8,&quot;expectedValue is &quot;:&quot;BACKEND_ADD&quot;,&quot;expectedValue is&quot;:1},&quot;createTime&quot;:{&quot;currentValue is &quot;:&quot;2019-07-05 11:03:31&quot;,&quot;expectedValue is &quot;:&quot;2020-01-16 09:12:12&quot;,&quot;expectedValue is&quot;:1},&quot;title&quot;:{&quot;currentValue is &quot;:&quot;&quot;,&quot;expectedValue is &quot;:&quot;test&quot;,&quot;expectedValue is&quot;:1},&quot;superId&quot;:{&quot;currentValue is &quot;:&quot;eef023f445604e7a92f25dd8c1b44c14&quot;,&quot;expectedValue is &quot;:&quot;47aa131dd5af48b59f51a588fb0d3a15&quot;,&quot;expectedValue is&quot;:1},&quot;projectName&quot;:{&quot;currentValue is &quot;:null,&quot;expectedValue is &quot;:&quot;新能源汽车大数据平台2&quot;,&quot;expectedValue is&quot;:1,&quot;actualValue&quot;:null,&quot;expectedValue&quot;:&quot;新能源汽车大数据平台2&quot;},&quot;userQuantity&quot;:{&quot;currentValue is &quot;:50,&quot;expectedValue is &quot;:&quot;新能源汽车大数据平台2&quot;,&quot;expectedValue is&quot;:5,&quot;actualValue&quot;:null,&quot;expectedValue&quot;:&quot;新能源汽车大数据平台2&quot;}}]">
      <![CDATA[java.lang.AssertionError: expected [{}] but found [{"id":{"currentValue is ":"96e1e2dd22b744e4978d76ac2b3f8cd3","expectedValue is ":"aa2a8a54936542b48e78346cb4f0df33"},"organization":{"currentValue is ":"北京理工新源信息有限公司","expectedValue is ":"新源科技"},"tenant":{"currentValue is ":"bitnei-cn","expectedValue is ":"bitnei-cn110222"},"connectMobile":{"currentValue is ":"18945695263","expectedValue is ":"18110024813"},"origin":{"currentValue is ":"REGISTER_FREE","expectedValue is ":"BACKEND_ADD"},"userNum":{"currentValue is ":8,"expectedValue is ":"BACKEND_ADD","expectedValue is":1},"createTime":{"currentValue is ":"2019-07-05 11:03:31","expectedValue is ":"2020-01-16 09:12:12","expectedValue is":1},"title":{"currentValue is ":"","expectedValue is ":"test","expectedValue is":1},"superId":{"currentValue is ":"eef023f445604e7a92f25dd8c1b44c14","expectedValue is ":"47aa131dd5af48b59f51a588fb0d3a15","expectedValue is":1},"projectName":{"currentValue is ":null,"expectedValue is ":"新能源汽车大数据平台2","expectedValue is":1,"actualValue":null,"expectedValue":"新能源汽车大数据平台2"},"userQuantity":{"currentValue is ":50,"expectedValue is ":"新能源汽车大数据平台2","expectedValue is":5,"actualValue":null,"expectedValue":"新能源汽车大数据平台2"}}]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.bitnei.apitest.testcases.saas.PostBackSaas.UserInfoQueryData(PostBackSaas.java:75)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- UserInfoQueryData -->
  <testcase name="UserInfoQueryData" time="0.203" classname="com.bitnei.apitest.testcases.saas.PostBackSaas"/>
  <testcase name="UserInfoQueryData" time="0.116" classname="com.bitnei.apitest.testcases.saas.PostBackSaas">
    <failure type="java.lang.AssertionError" message="expected [{}] but found [{&quot;size&quot;:{&quot;currentValue is &quot;:0,&quot;expectedValue is&quot;:1},&quot;pages&quot;:{&quot;currentValue is &quot;:0,&quot;expectedValue is&quot;:1},&quot;total&quot;:{&quot;currentValue is &quot;:0,&quot;expectedValue is&quot;:1},&quot;Length&quot;:{&quot;actualLenth&quot;:0,&quot;expectLenth&quot;:1}}]">
      <![CDATA[java.lang.AssertionError: expected [{}] but found [{"size":{"currentValue is ":0,"expectedValue is":1},"pages":{"currentValue is ":0,"expectedValue is":1},"total":{"currentValue is ":0,"expectedValue is":1},"Length":{"actualLenth":0,"expectLenth":1}}]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.bitnei.apitest.testcases.saas.PostBackSaas.UserInfoQueryData(PostBackSaas.java:75)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- UserInfoQueryData -->
  <testcase name="UserInfoQueryData" time="0.166" classname="com.bitnei.apitest.testcases.saas.PostBackSaas">
    <failure type="java.lang.AssertionError" message="expected [{}] but found [{&quot;size&quot;:{&quot;currentValue is &quot;:0,&quot;expectedValue is&quot;:1},&quot;pages&quot;:{&quot;currentValue is &quot;:0,&quot;expectedValue is&quot;:1},&quot;total&quot;:{&quot;currentValue is &quot;:0,&quot;expectedValue is&quot;:1},&quot;Length&quot;:{&quot;actualLenth&quot;:0,&quot;expectLenth&quot;:1}}]">
      <![CDATA[java.lang.AssertionError: expected [{}] but found [{"size":{"currentValue is ":0,"expectedValue is":1},"pages":{"currentValue is ":0,"expectedValue is":1},"total":{"currentValue is ":0,"expectedValue is":1},"Length":{"actualLenth":0,"expectLenth":1}}]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.bitnei.apitest.testcases.saas.PostBackSaas.UserInfoQueryData(PostBackSaas.java:75)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- UserInfoQueryData -->
  <testcase name="UserInfoQueryData" time="0.152" classname="com.bitnei.apitest.testcases.saas.PostBackSaas"/>
</testsuite> <!-- com.bitnei.apitest.testcases.saas.PostBackSaas -->
